test/metric/metric_value_query.go

```diff
@@ -21,10 +21,19 @@ import (
 type MetricValueFetcher struct {
 }
 
+func logDimensions(dims []types.Dimension) {
+	log.Printf("\tDimensions:\n")
+	for _, d := range dims {
+		if d.Name != nil && d.Value != nil {
+			log.Printf("\t\tDim(name=%q, val=%q\n", *d.Name, *d.Value)
```

Comment:
I think that we could use a tab writer to pretty print these instead but I won't block on it

Reply:
I'd be fine removing it too.
I'd like to put it off since there are a lot of useless prints in this repository that should be reviewed.
I see a bunch of memory addresses (aka pointers) which I do not find useful.